controllers/autoneg.go

```diff
@@ -258,15 +258,10 @@ func (b *ProdBackendController) ReconcileBackends(actual, intended AutonegStatus
 							var syncConfig AutonegSyncConfig = *intended.AutonegSyncConfig
 							if syncConfig.CapacityScaler != nil && *syncConfig.CapacityScaler == true {
 								be.CapacityScaler = u.CapacityScaler
-								forceCapacity[beidx] = true
 							}
 						} else {
 							// Force CapacityScaler to an "empty value"
 							u.CapacityScaler = 0
-							// Check if existing capacity scaler is zero
-							if be.CapacityScaler == 0 {
-								forceCapacity[beidx] = true
```

Comment:
Might this work just by adding forceCapacity to true anyway without the check. Or is my thinking wrong here.

Reply:
Do you mean dropping `AutonegSyncConfig` completely?